packages/suite-web/e2e/tests/wallet/send-form-ltc.test.ts

```diff
@@ -11,8 +13,11 @@ describe('LTC send form with mocked blockbook', () => {
         cy.task('startBridge');
 
         cy.viewport(1440, 2560).resetDb();
-        cy.prefixedVisit('/settings/coins');
```

Comment:
btw navigating directly to an url doesn't work anymore? I  am seeing so many changes of this type.